src/Sulu/Bundle/PageBundle/Resources/js/views/PageList/PageList.js

```diff
@@ -53,20 +54,28 @@ class PageList extends React.Component<Props> {
             return;
         }
 
-        if (webspace.allLocalizations.find((localization) => localization.localization === userStore.contentLocale)) {
-            this.locale.set(userStore.contentLocale);
+        let locale = null;
```

Comment:
Would prefer to make this a `const`, to avoid having troubles by somebody reassigning this.

```javascript
const locale = webspace.allLocalizations.find((localization) => localization.localization === userStore.contentLocale)
    ? userStore.contentLocale
    : this.findDefaultLocale(webspace.localizations);
```

Reply:
Changed!